src/theme/components/hero.ts

```diff
@@ -156,18 +156,16 @@ const primary = definePartsStyle(({ foregroundColor, isDarkText }) => ({
     backgroundSize: "cover",
     backgroundPosition: "center",
     display: "grid",
-    gridTemplateColumns: { base: "100%", md: "repeat(6, 1fr)" },
-    gridTemplateAreas: {
-      base: `"content"`,
-      md: `".  content content content content ."`,
-    },
+    gridTemplateColumns: "repeat(12, 1fr)",
+    gap: { base: "1.5rem", lg: "2rem" },
     minHeight: "352px",
     p: "l",
+    paddingInline: "1rem",
   },
   content: {
     bg: "ui.black",
     color: getTextColor("body", "light", foregroundColor, isDarkText),
-    gridArea: "content",
+    gridColumn: { base: "1 / -1", md: "2 / 12", lg: "3 / 11" },
     maxWidth: { md: "860px" },
```

Comment:
This can be removed. Yes, it allows the width of the content area to be greater than the `800px` max width recommendation, but that's ok.